public/js/pimcore/object/tree.js

```diff
@@ -1119,7 +1119,7 @@ pimcore.registerNS("pimcore.object.tree");
              elementType: "object",
              elementSubType: record.data.type,
              id: record.data.id,
-             default: Ext.util.Format.htmlDecode(record.data.text)
+             default: record.data.key
```

Comment:
please revert as this is required https://github.com/pimcore/pimcore/pull/5206/files#diff-9d9bcbb5d9a089108c6e32c3b3885884382b70ce63550e82ddebbc81d33a39eeR310

Reply:
done with 7131182ed24c7c1a40acce25e2fabc411a298da7